src/pages/Dashboard/Schedule/events/colours.js

```diff
@@ -0,0 +1,17 @@
+export const HTV_OFFICIAL = {
+    background: '#7bbe48',
+    primary: 'white',
+    secondary: 'white',
+};
+
+export const WORKSHOPS = {
+    background: '#c29356',
+    primary: 'white',
+    secondary: 'white',
+};
+
+export const SIDE = {
+    background: '#236fc2',
+    primary: 'white',
+    secondary: 'white',
+};
```

Comment:
I copied the files in Schedule/events from last year HTV to use as test data, https://github.com/fpunny/htv-live/tree/master/src/schedule.
When adding the actual data for the schedule, we will need to update the colours and friday/saturday/sunday files